data/templates/install-ubuntu/post-install-ubuntu.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # create SSH key for root
-<% if (undefined !== rootSshKey) { -%>
+<% if ('undefined' !== typeof rootSshKey && null !== rootSshKey) { -%>
```

Comment:
Thanks @benbp. I know this piece looks very clumsy. 
Using `!=` looks good, but it breaking our convention, doesn't it ? 
I think you mean `if (this.hasOwnProperty("rootSshKey"))`, right ? This should work but I still need to check `null` value.
The clear way maybe using `_.isEmpty(this.rootSshKey)`, but unfortunately, we can not using lodash directly in ejs , right ?